tests/utils/session.py

```diff
@@ -33,7 +39,8 @@ def __eq__(self, other) -> bool:
         return self.method == other.method and \
             self.path == other.path and \
             self.json == other.json and \
-            self.params == other.params
+            self.params == other.params and \
+            (not self.version or not other.version or self.version == other.version) # Allows users to check the URL version without forcing everyone to.
```

Comment:
```suggestion
            (self.version is None or other.version is None or self.version == other.version) # Allows users to check the URL version without forcing everyone to.
```
Many more things evaluate to False than `None`.

Are you sure it should pass if one case has a None and the other does not?

Reply:
Correct, and the only time they should be checked is if they're truth-y. Which is why the recommendation is to prefer truthiness checking over explicit `None` checking if at all possible.

And yes. Failing if only one is set means that every time someone performs a check against `FakeCall` they must specify the version, which is onerous since in many case they don't care. But it allows tests which do care to perform such a check.